### PR TITLE
[circle-mpqsolver] Tidy Quantizer

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit replaces implementation of `quantize`
by another `quantize`.

This commit was tested in https://github.com/Samsung/ONE/pull/11849

Draft: https://github.com/Samsung/ONE/pull/11849
Related: https://github.com/Samsung/ONE/issues/11374

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by jinevening — DISMISSED** [review]
LGTM

File: compiler/circle-mpqsolver/src/core/Quantizer.cpp [hunk 1]
```diff
@@ -114,59 +114,9 @@ bool Quantizer::quantize(luci::Module *module, const std::string &quant_dtype,
  * @brief quantize recorded module (min/max initialized) with specified parameters
  * returns true on success
  */
-
```
Comment (seanshpark) [inline compiler/circle-mpqsolver/src/core/Quantizer.cpp:117]: plz move this to another PR

Comment (stamalakhov) [reply compiler/circle-mpqsolver/src/core/Quantizer.cpp:117]: Sorry. Fixed.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
